examples/tsconfig.json

```
@@ -0,0 +1,13 @@
+{
+  "compilerOptions": {
+    "target": "es6",
+    "module": "es6",
+    "lib": ["es2017", "es7", "es6", "dom"],
```

```suggestion
    "target": "esnext",
    "module": "esnext",
    "lib": ["esnext", "dom"],
```

maybe a more modern solution? (iirc this is right but haven't checked)